sdk/nodejs/policy/server.ts

```diff
@@ -48,14 +46,18 @@ import { version } from "./version";
 
 // ------------------------------------------------------------------------------------------------
 
-let serving = false;
+// Name of the policy pack currently being served, if applicable.
+let servingPolicyPack: string | undefined = undefined;
 
 export function serve(policyPackName: string, policyPackVersion: string, policies: Policies): void {
-    if (serving !== false) {
-        throw Error("Only one policy gRPC server can run per process");
+    if (servingPolicyPack) {
```

Comment:
We don't currently do any validation of `policyPackName` in this library. We probably should just do the same validation that the service does inside the `PolicyPack` constructor. Otherwise, right now, I could pass `undefined`, `null`, or `""`, which would throw off this logic.

Reply:
SGTM, I'll send that out in a separate PR.